contracts/upgradeable_contracts/multi_amb_erc20_to_erc677/MultiTokenLanePermissions.sol

```diff
@@ -0,0 +1,116 @@
+pragma solidity 0.4.24;
+
+import "../../upgradeable_contracts/Ownable.sol";
+
+/**
+ * @title MultiTokenLanePermissions
+ * @dev Multi token mediator functionality for managing destination AMB lanes permissions.
+ */
+contract MultiTokenLanePermissions is Ownable {
+    uint256 internal constant TOKEN_DESTINATION_LANE_TAG = 0;
```

Comment:
could you confirm that these constant are not held on the storage so we don't need to assume a special procedure to initialize them as part of the contracts upgrade procedure.

Reply:
Yes, I can confirm that, such internal constants go directly to the bytecode, nothing special is needed to use them under a proxy contract. In other contracts, we have similar constants as well: `MAX_FEE`, `MAX_VALIDATORS`, etc..

Reply:
thanks